base/serialize.jl

```diff
@@ -102,14 +102,20 @@ function serialize_cycle(s::SerializationState, x)
     return false
 end
 
-serialize(s::SerializationState, x::Bool) = x ? writetag(s.io, TRUE_TAG) :
+function reset(s::AbstractSerializer)
```

Comment:
Binding `reset` already exists and is exported. Reusing it instead of exporting a new one.